test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_core_test.clj

```diff
@@ -189,125 +192,139 @@
+      (update-in [:https :protocols] identity)
+      (update-in [:https :cipher-suites] identity)
+      (update-in [:https :client-auth] (fnil identity :none))
+      (update-in [:https :keystore-config]
+                 (fnil identity
+                       {:truststore (-> (KeyStore/getDefaultType)
+                                        (KeyStore/getInstance))
+                        :key-password "hello"
+                        :keystore (-> (KeyStore/getDefaultType)
+                                      (KeyStore/getInstance))}))))
+
+(defn create-server-with-config
+  [config]
+  (jetty/create-server (get-webserver-context-for-server) config))
+
+(defn create-server-with-partial-http-config
+  [config]
+  (create-server-with-config (munge-http-connector-config config)))
+
+(defn create-server-with-partial-http-and-https-config
+  [config]
+  (create-server-with-config (munge-http-and-https-connector-config config)))
+
+(defn get-thread-pool-for-partial-http-config
+  [config]
+  (.getThreadPool (create-server-with-partial-http-config config)))
+
+(defn get-thread-pool-for-default-server
+  []
+  (.getThreadPool (Server.)))
+
+(defn default-server-max-threads
+  []
+  (.getMaxThreads (get-thread-pool-for-default-server)))
+
+(defn get-max-threads-for-partial-http-config
+  [config]
+  (.getMaxThreads (get-thread-pool-for-partial-http-config config)))
+
+(deftest create-server-max-threads-test
+  (testing "default max threads passed through to thread pool"
+    (is (= (default-server-max-threads)
+           (get-max-threads-for-partial-http-config {:max-threads nil}))))
+  (testing "custom max threads passed through to thread pool"
+    (is (= 9042
+           (get-max-threads-for-partial-http-config
+             {:max-threads 9042, :queue-max-size nil})))))
+
+(deftest create-server-queue-max-size-test
+  (let [get-queue-for-partial-http-config (fn [config]
+                                            (get-server-thread-pool-queue
+                                              (create-server-with-config
+                                                (munge-http-connector-config
+                                                  config))))
+        default-server-min-threads        (.getMinThreads
+                                            (get-thread-pool-for-default-server))]
+    (testing "default queue max size passed through to thread pool queue"
+      (is (= (.getMaxCapacity (get-server-thread-pool-queue (Server.)))
+             (.getMaxCapacity (get-queue-for-partial-http-config
+                                {:queue-max-size nil})))))
+    (testing "custom default queue max size passed through to thread pool queue"
+      (is (= 393
+             (.getMaxCapacity (get-queue-for-partial-http-config
+                                {:queue-max-size 393})))))
+    (testing (str "default max threads passed through to thread pool when "
+                  "queue-max-size set")
+      (is (= (default-server-max-threads)
```

Comment:
This looks like a bug - looks like accidental parens around `default-server-min-threads` - that seems like it should throw an error since `default-server-min-threads` should be an integer, right?